slack-go.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+define("GOSERVER", "http://go.brandwatch.com/");
+define("GOSERVERSHORT", "http://go/");
+define("GOSERVERDISPLAY", "go");
```

Comment:
Could we make this `Go/`? I would like the official title to have a capital and the slash everywhere.